src/authorization.ts

```diff
@@ -28,15 +28,15 @@ import {
 
 import {
   sha512_256
-} from './vendor/js-sha512';
```

Comment:
The reason we didn't use a direct import is because we're not too sure about the published package, maintainers and future updates. Do you think it's safe to directly import?

Reply:
Ah I see. I updated it to use the existing `sha.js` lib and removed that dependency. It was missing the `"SHA-512/256"` IV but was easy to specify.